qiskit/algorithms/quantum_time_evolution/variational/principles/variational_principle.py

```diff
@@ -43,28 +43,30 @@ def _lazy_init(
         self,
         hamiltonian,
         ansatz,
-        param_dict: Dict[Parameter, Union[float, complex]],
-        regularization: Optional[str] = None,
+        parameters: List[Parameter],
+        # regularization: Optional[str] = None,
     ):
+
         self._hamiltonian = hamiltonian
         self._ansatz = ansatz
-        self._param_dict = param_dict
         self._operator = ~StateFn(hamiltonian) @ StateFn(ansatz)
-        self._operator = self._operator / self._operator.coeff  # Remove the time from the operator
-        raw_metric_tensor = self._get_raw_metric_tensor(ansatz, param_dict)
+        self._params = parameters
```

Comment:
Let's unify names.

Reply:
You can choose whatever you want. I just introduced to avoid that the `param_dict` is passed all the time.